ember.s

```diff
@@ -1,26 +1,35 @@
-; https://github.com/NieDzejkob/ember
-; this overwrites bootOS :D
-; you can install it to you hdd too
-
 ; commands to install:
 ;   * run 00<your drive id here>d
-;   * run 7c00w00000000
+;   * run 7a00w00000000
 ;     * installs it to the first sector
 
 ; memory map:
 ; [0000; 0400) IVT
 ; [0400; 0500) BDA
-; [    ; 7c00) stack
-; [7c00; 7e00) MBR
-; [7de0; 7deb) filename buffer
-; [7df0; 7e00) EDD disk packet
-; [7e00;     ) variables
+; [    ; 7a00) stack
+; [7a00; 7c00) MBR
+; [7be0; 7beb) filename buffer
+; [7e00; 7e10) EDD disk packet
+; [7e10;     ) variables
 
-org 0x7c00
+org 0x7a00
 bits 16
 
 %define linebuffer 0x600
 ; Some BIOSes start with CS=07c0. Make sure this does not wreak havoc
+	cli
+	cld
+	xor cx, cx
+	mov ds, cx
+	mov es, cx
+	mov ss, cx
+	mov sp, 0x7a00
+
+	mov si, 0x7c00
+	mov di, 0x7a00
+	mov cx, 512
+	rep movsb
+
```

Comment:
What motivates this relocation?

Reply:
It's useful so I can load programs from disk to 0x7c00 and execute them